k2/csrc/fsa_util.cc

```diff
@@ -15,14 +15,13 @@ void GetEnteringArcs(const Fsa &fsa, std::vector<int32_t> *arc_index,
                      std::vector<int32_t> *end_index) {
   // CHECK(CheckProperties(fsa, KTopSorted));
 
-  int32_t num_states = fsa.NumStates();
+  auto num_states = fsa.NumStates();
   std::vector<std::vector<int32_t>> vec(num_states);
-  int32_t num_arcs = 0;
+  auto num_arcs = static_cast<int32_t>(fsa.arcs.size());
```

Comment:
I think this variable isn't needed any more since it's only accessed once; could just use fsa.arcs.size().

Reply:
I have changed this in previous PR.

Reply:
cool.  BTW there are conflicts now.  I'll look at this again when you remove WIP.

Reply:
conflicts resolved.